firefox-ios/Shared/Strings.swift

```diff
@@ -2333,6 +2333,24 @@ extension String {
             value: "Google Search",
             comment: "When making a new search from the awesome bar, search results appear as the user write new letters in their search. This string will be used as a header for Google search results listed as suggestions.")
     }
+
+    public struct SearchZero {
+        public static let RecentSearchesSectionTitle = MZLocalizedString(
+            key: "SearchZero.RecentSearches.SectionTitle.v146",
+            tableName: "SearchZero",
+            value: "Recent Searches",
+            comment: "When user taps on the address bar, the user enters the search zero view. The user can see their recent searches in this view. This is the title of the section that shows the list of their recent searches.")
```

Comment:
I would get rid of this part "he user enters the search zero view" since it might be obscure to localizers, and just make it part of the rest of the sentences.
```suggestion
            comment: "When a user taps on the address bar, they can see their recent searches. This is the title of the section that shows the list of their recent searches.")
```